squad-server/layers/layer.js

```diff
@@ -31,7 +31,7 @@ export default class Layer {
           respawnDelay: vehicle.respawnTime
         })),
         numberOfTanks: (data[t].vehicles || []).filter((v) => {
-          return v.icon.match(/tank/);
+          return v.icon.match(/_tank/);
```

Comment:
Why is this in this PR?

Reply:
I pushed the commits on the wrong branch, now it should be fixed.